doc/http-api-guide.md

```diff
@@ -29,10 +29,11 @@ available in the future.
 [Developer's Guide]: http://www.bloomberglabs.com/files/2014/07/blpapi-developers-guide-2.54.pdf
 
 
-Requesting data
----------------
+Request/Response data
```

Comment:
OK, perhaps a better question is: should we switch completely to # for headings, rather than mixing underlines and #? :)